### PR TITLE
pidgin: update to 2.14.6

### PR DESCRIPTION
**Pull request by tibequadorian**
Continuation of #27704

`-Ddoc=false` doesn't work because the option doesn't exist :/ @ericonr 

#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/libgnt/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+build_helper=gir
+hostmakedepends="pkg-config glib-devel gtk-doc"
+makedepends="libxml2-devel ncurses-devel python3-devel glib-devel gtk-doc"
+short_desc="GLib Ncurses Toolkit"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="GPL-2.0-or-later"
+homepage="https://keep.imfreedom.org/libgnt/"
+distfiles="${SOURCEFORGE_SITE}/pidgin/libgnt/${version}/libgnt-${version}.tar.xz"
+checksum=61cf74b14eef10868b2d892e975aa78614f094c8f4d30dfd1aaedf52e6120e75
+patch_args=" -Np1"
```
Comment (ericonr) [inline srcpkgs/libgnt/template:15]: No longer necessary, it's now the default.

File: srcpkgs/libgnt/template [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'libgnt'
+pkgname=libgnt
+version=2.14.2
+revision=1
+build_style=meson
+build_helper=gir
+hostmakedepends="pkg-config glib-devel gtk-doc"
+makedepends="libxml2-devel ncurses-devel python3-devel glib-devel gtk-doc"
+short_desc="GLib Ncurses Toolkit"
```
Comment (ericonr) [inline srcpkgs/libgnt/template:9]: Wow, the existence of such a thing hurts me :D

File: srcpkgs/libgnt/template [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+pre_configure() {
+	# disable doc for cross builds
+	if [ "$CROSS_BUILD" ]; then
+		vsed -i "/subdir('doc')/d" meson.build
+	fi
+}
+
+libgnt-devel_package() {
+	depends="libglib-devel libgnt>=${version}_${revision}"
+	short_desc="IM library extracted from Pidgin (GNT) - development files"
```
Comment (ericonr) [inline srcpkgs/libgnt/template:26]:
```suggestion
	short_desc+=" - development files"
```

But maybe fix the main `short_desc` to match whatever you prefer?

File: srcpkgs/libgnt/template [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+	pkg_install() {
+		vmove usr/include/gnt
+		vmove usr/lib/pkgconfig/gnt.pc
```
Comment (ericonr) [inline srcpkgs/libgnt/template:29]:
```suggestion
		vmove usr/include
		vmove usr/lib/pkgconfig
```

**Review by ericonr — COMMENTED** [review]
Thanks for picking this up!

File: srcpkgs/libgnt/template (hunk 3 above)
Comment (tibequadorian) [reply srcpkgs/libgnt/template:26]: oops

**Pull request merged**